eng/helix/helix.proj

```diff
@@ -12,6 +12,20 @@
   <Import Project="..\Build.props" />
   <Import Project="..\Versions.props" />
 
+  <!-- Ensure shared framework is available for correlation payload. -->
+  <ItemGroup Condition=" '$(DoNotRequireSharedFxHelix)' != 'true' " >
+    <ProjectReference Include="$(RepoRoot)\src\Framework\App.Ref\src\Microsoft.AspNetCore.App.Ref.csproj"
```

Comment:
I wonder why these are not referenced in the wixproj's for the installers instead of it trying to (and failing) to unzip the files to a zip file that does not exists (which I think could be avoided if they referenced these as well).